app/controllers/comments_controller.rb

```diff
@@ -1,2 +1,18 @@
 class CommentsController < ApplicationController
```

Comment:
Add authorizations in comments policy + in methods

Reply:
done